client/components/SmallRatingStar.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+class SmallRatingStar extends React.Component {
+  render() {
+    return (
+      <div className="review-item-rating">
+        <div className="rating-star-small">
+
+          <div className="icon-small-star-full">
+            <svg className="small-icon" viewBox="0 0 23 23" width="13" height="13">
+              <polygon fill="#212a2f" points="23,8.8 15,7.6 11.5,0 8,7.6 0,8.8 5.7,14.7 4.4,23 11.5,19 18.6,23 17.2,14.7"></polygon>
+            </svg>
+          </div>
+
+          <div className="icon-small-star-full">
+            <svg className="small-icon" viewBox="0 0 23 23" width="13" height="13">
+              <polygon fill="#212a2f" points="23,8.8 15,7.6 11.5,0 8,7.6 0,8.8 5.7,14.7 4.4,23 11.5,19 18.6,23 17.2,14.7"></polygon>
+            </svg>
+          </div>
+
+          <div className="icon-small-star-full">
+            <svg className="small-icon" viewBox="0 0 23 23" width="13" height="13">
+              <polygon fill="#212a2f" points="23,8.8 15,7.6 11.5,0 8,7.6 0,8.8 5.7,14.7 4.4,23 11.5,19 18.6,23 17.2,14.7"></polygon>
+            </svg>
+          </div>
+
+          <div className="icon-small-star-full">
+            <svg className="small-icon" viewBox="0 0 23 23" width="13" height="13">
+              <polygon fill="#212a2f" points="23,8.8 15,7.6 11.5,0 8,7.6 0,8.8 5.7,14.7 4.4,23 11.5,19 18.6,23 17.2,14.7"></polygon>
+            </svg>
+          </div>
+
+          <div className="icon-small-star-full">
+            <svg className="small-icon" viewBox="0 0 23 23" width="13" height="13">
+              <polygon fill="#212a2f" points="23,8.8 15,7.6 11.5,0 8,7.6 0,8.8 5.7,14.7 4.4,23 11.5,19 18.6,23 17.2,14.7"></polygon>
+            </svg>
+          </div>
+
+        </div>
+      </div>
+    );
+  }
+}
+
```

Comment:
This should definitely be dynamically rendered. It's fine to shim it because our product page has almost uniquely positive reviews, but take a look at the star renderer in ProductOptions if you'd like to see an idea of how this would work.

Reply:
I will, thank you!